# This file hold all the configuration parameters specifically
# relating to the forklift and its dimensions/specifications
# All units should be in SI unless otherwise specified

# General Dimensions
forklift:
    base_width: 0.400
    
# Frame transform parameters
front_axel_middle:
    pose:
        x: 0.0
        y: 0.0
        z: 0.0
        qx: 0.0
        qy: 0.0
        qz: 0.0
        qw: 1.0
    frames:
        frame_id: "base_link"
        child_frame_id: "front_axel_middle_link"
        
wheel_left:
    pose:
        x: 0.0
        y: 0.200
        z: 0.0
        qx: 0.0
        qy: 0.0
        qz: 0.0
        qw: 1.0
    frames:
        frame_id: "front_axel_middle_link"
        child_frame_id: "wheel_left_link"

wheel_right:
    pose:
        x: 0.0
        y: -0.200
        z: 0.0
        qx: 0.0
        qy: 0.0
        qz: 0.0
        qw: 1.0
    frames:
        frame_id: "front_axel_middle_link"
        child_frame_id: "wheel_right_link"
